sdk/src/main/java/io/customer/sdk/CustomerIO.kt

```diff
@@ -141,7 +141,8 @@ class CustomerIO internal constructor(
                 urlHandler = urlHandler,
                 autoTrackScreenViews = shouldAutoRecordScreenViews,
                 backgroundQueueMinNumberOfTasks = 10,
-                backgroundQueueSecondsDelay = 30.0
+                // TODO put this value back to 30
+                backgroundQueueSecondsDelay = 10.0
```

Comment:
was this meant to be pushed? commenting for your visibility

Reply:
fixed. thanks for the reminder.